--- Mongoose-2.0.4/CMakeLists.txt.orig	2023-01-16 20:09:13.319621223 +0100
+++ Mongoose-2.0.4/CMakeLists.txt	2023-01-16 20:51:05.869342906 +0100
@@ -174,7 +174,7 @@ set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMA
 set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
 
 # Build the Mongoose library
-add_library(mongoose_lib ${MONGOOSE_LIB_FILES})
+add_library(mongoose_lib STATIC ${MONGOOSE_LIB_FILES})
 set_property(TARGET mongoose_lib PROPERTY POSITION_INDEPENDENT_CODE ON)
 set_target_properties(mongoose_lib
         PROPERTIES OUTPUT_NAME mongoose)
